examples/sendingDomains/delete_sendingDomain.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var key = 'YOURAPIKEY'
+  , SparkPost = require('sparkpost')
+  , client = new SparkPost(key);
+
+client.sendingDomains['delete']('example1.com', function(err, data) {
```

Comment:
It is everywhere. I'll fix it in this PR. I'll create an issue to clean that up as part of v2.0.0